README.md

```diff
@@ -1 +1,54 @@
-# project-capstone
\ No newline at end of file
+# Capstone Project
+
+> This project contains the HTML, CSS and JavaScript Capstone Project. 
+
+
+# Project Name
+## THE PAKWHEEL show 2022
+
+**   PakWheels has been hosting auto shows in Karachi since 2015. Every year, we arrange this exciting event and call out all car lovers. An auto show, also known as a motor show or car show, is a public exhibition of current automobile models, debuts, concept cars, or out-of-production classics. It is attended by automotive industry representatives, dealers, auto journalists and car enthusiasts. Most auto shows occur once or twice a year** 
+# WORK FLOW SCREEN 
+## DESKTOP VIEW
+![screenshot](./img/1ss.png)
+## MOBILE VIEW
+![screenshot](./img/4ss.png)
+## Video Presentation
+[Click Here to view presentation video](https://www.loom.com/share/7d2164812b454565bd1fe06c0e7b7df6)
+## Live Demo
+
```

Comment:
- [ ] It looks like you forgot to add the link to live project, please kindly add the link here ☀️